packages/core/src/dispatcher.ts

```diff
@@ -151,6 +151,10 @@ export class Dispatcher {
           })
         }
 
+        // put the payload into ipfs
+        jws.link = await this._shutdownSignal.abortable((signal) => {
```

Comment:
what does the change to this file do?  It looks like it's just moving the storing of the payload earlier?  Is there a reason for that change?

Reply:
Oh I guess there's actually two changes.  Moving the block earlier, and changing it to override jws.link with the result.  Moving the block earlier in the function made me miss the functional change.